### ISSUE TITLE
Ksads grades in school error?

### ISSUE THREAD

**Issue by cpalmer14** [ABCD-STUDY/DEAP#81]
Hi There are 73 people that have a grade of '-1' but I'm not sure what this means?  See image below from Explore in DEAP.

![image](https://user-images.githubusercontent.com/44177108/48725194-405f7700-ebe0-11e8-921d-468eea523ce4.png)

**Comment by cpalmer14** [comment]
I think they're meant to be NAs

**Comment by HaukeBartsch** [comment]
The issue there is that we receive the data from KSADS. We will need to ask them what the meaning is of these values - hopefully its as simple as converting them to NA. Do you know more?

**Comment by cpalmer14** [comment]
I've just noticed in the description on Explore on DEAP it says that -1 should be NA but I don't know what the other NAs are.

![image](https://user-images.githubusercontent.com/44177108/48726411-1d36c680-ebe4-11e8-958e-629d26f80514.png)

**Comment by HaukeBartsch** [comment]
The other NA's are for missing data. The difference there is that -1 equals a not applicable based on previous questions. Do you have a suggestion for how to make this easier to understand for users?

**Comment by cpalmer14** [comment]
Can you maybe just highlight that in the description below the data summary in DEAP?  E.g. "-1 = Not applicable (participant selected NA/didn't answer this question when completing KSADS); NA = missing data".  However, the number of NAs appears to be different across questions in the KSADS.  If it's people who didn't complete the KSADS at all for whatever reason I'm not sure why the number of NAs isn't the same across the questionnaire? I'm not very familiar with how it appears to the participant but I can someone who administers this

**Comment by HaukeBartsch** [comment]
Yes, I see. The problem is that different portions of the data can be missing. KSADS administers them in some order and the participant can decide to stop the interview at any point. Its not that all the data will be missing if there is any NA.

**Comment by cpalmer14** [comment]
Ah of course.  Important just to distinguish between what the two different types of NA are then

**Comment by HaukeBartsch** [comment]
So, we could leave the data and the data descriptions as is? Might be good to change the "Not applicable" to "Not applicable due to previous answers".

**Comment by cpalmer14** [comment]
I just spoke with Vanessa and Margie and these questions are actually asked in Redcap and Not Applicable is an option so the -1 means the parent has selected "Not Applicable" as their answer.  Maybe the description could read: "-1 = Not Applicable (selected by parent); NA= missing data" just to make it really clear for external users.
